vscode-extension/src/createLoopForm.html

```diff
@@ -78,12 +78,17 @@ <h2>Create Loop</h2>
         </span>
       </div>
       <div class="pam">
-        <label for="path"><span class="required">*</span>Project Path</label><br />
+        <label for="path"><span class="required">*</span>Project Path</label
+        ><br />
         <input type="text" id="path" name="path" value="" />
-        <button class="pas" onclick="openFolderBrowser()" type="button">Browse</button>
+        <button class="pas" onclick="openFolderBrowser()" type="button">
+          Browse
+        </button>
       </div>
       <div class="pam">
-        <label for="projectName"><span class="required">*</span>Project Name</label><br />
+        <label for="projectName"
+          ><span class="required">*</span>Project Name</label
+        ><br />
```

Comment:
I won't hold up the PR over this but some of these HTML tags got messed up like
```html
  </label
>
```

Reply:
This should have been addressed now. Though, it looks like some of those may have already been there